ocs_ci/ocs/node.py

```diff
@@ -1073,3 +1073,64 @@ def generate_node_names_for_vsphere(count, prefix="compute-"):
         f"{prefix}{current_compute_suffix + node_count}"
         for node_count in range(1, count + 1)
     ]
+
+
+def check_taint_on_nodes(taint=None):
+    """
+    Function to check for particular taint on nodes
+
+    Args:
+        taint (str): The taint to check on nodes
+
+    """
+    taint = taint if taint else "node.ocs.openshift.io/storage=true:NoSchedule"
+    ocs_nodes = get_ocs_nodes()
+    flag = -1
+    for node_obj in ocs_nodes:
+        if node_obj.get().get("spec").get("taints"):
+            if node_obj.get().get("spec").get("taints")[0].get("key") in taint:
+                log.info(f"Node {node_obj.name} has taint {taint}")
+                flag = 1
+        else:
+            flag = 0
+    return bool(flag)
+
+
+def taint_nodes(nodes_to_taint=None):
+    """
+    Function to taint nodes
+
+    Args:
+        nodes_to_taint (list): Nodes to taint
+
+    """
+    if not check_taint_on_nodes():
+        ocp = OCP()
+        ocs_nodes = get_ocs_nodes()
+        nodes_to_taint = nodes_to_taint if nodes_to_taint else ocs_nodes
+        log.info(f"Taint nodes with taint: " f"{constants.OPERATOR_NODE_TAINT}")
+        for node in nodes_to_taint:
+            taint_cmd = f"adm taint nodes {node.name} {constants.OPERATOR_NODE_TAINT}"
+            ocp.exec_oc_cmd(command=taint_cmd)
+    else:
+        log.info(f"Nodes already have taint {constants.OPERATOR_NODE_TAINT} ")
+
+
+def untaint_nodes(taint=None, nodes_to_taint=None):
+    """
+    Function to remove taints from nodes
+
+    Args:
+        taint (str): taint to use
+        nodes_to_taint (list): list of nodes to untaint
+
+    """
+    if check_taint_on_nodes():
+        ocp = OCP()
+        ocs_nodes = get_ocs_nodes()
+        taint = taint if taint else constants.OPERATOR_NODE_TAINT
```

Comment:
consider setting the `taint` function arg with a default value of `constants.OPERATOR_NODE_TAINT`.
Then you will not need this line